src/tests/backend/specs/enforceLowerCasePadIds.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const assert = require('assert').strict;
+const common = require('../common');
+const padManager = require('../../../node/db/PadManager');
+const settings = require('../../../node/utils/Settings');
+
+describe(__filename, function () {
+  this.timeout(30000);
```

Comment:
Do the tests time out if this is removed? If not, I prefer to leave out the timeout call. If so, I'd prefer to move the timeout to the specific test case or hook that causes the timeout.

Reply:
no this seems to be a copy and paste error, see 21d3a1508a57be98cf26767d5b30ca25d9b18115